### PR TITLE
[Improvement-14146][K8S Task] print complete error logs in task log file

### LINKED ISSUE #14146: [Improvement][K8S Task] print complete error logs in task log file
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

If execution of K8S Task meets some exception, we can not get more exception information from current task log file.

image

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Radeity**
## Purpose of the pull request
- This PR closes https://github.com/apache/dolphinscheduler/issues/14146

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Comment by Radeity** [comment]
Maybe this PR can be labeled `3.1.7`

File: dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/k8s/AbstractK8sTask.java [hunk 1]
```diff
@@ -48,8 +48,9 @@ public void handle(TaskCallBack taskCallBack) throws TaskException {
             setExitStatusCode(response.getExitStatusCode());
             setAppIds(response.getAppIds());
         } catch (Exception e) {
+            log.error("k8s task submit failed with error", e);
             exitStatusCode = -1;
-            throw new TaskException("k8s process failure", e);
+            throw new TaskException("Execute k8s task error", e);
```
Comment (fuchanghai) [inline dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/k8s/AbstractK8sTask.java:53]: hi @Radeity  do we need lowercase ```Execute```?

**Comment by Radeity** [comment]
> hi @Radeity do we need lowercase Execute?

Hi, @fuchanghai , all other types of task are upper case, unless we modify them all.

**Review by fuchanghai — APPROVED** [review]
LGTM

**Review by caishunfeng — APPROVED** [review]

**Comment by caishunfeng** [comment]
> Maybe this PR can be labeled `3.1.7`

Ok, I have added the label.

**Pull request merged**